ingest-app/src/cmr/ingest/services/granule_bulk_update_service.clj

```diff
@@ -28,9 +28,34 @@
 (def granule-bulk-update-schema
  (js/json-string->json-schema (slurp (io/resource "granule_bulk_update_schema.json"))))
 
-(defn- validate-granule-bulk-update
- [json]
- (js/validate-json! granule-bulk-update-schema json))
+(defn- validate-granule-bulk-update-json
+  "Validate the request against the schema and return the parsed request."
+  [json]
+  (js/validate-json! granule-bulk-update-schema json))
+
+(defn- validate-granule-bulk-update-job-name
```

Comment:
Do we have this requirement to enforce unique job name within a provider? From what @mschmele  coded, he is actually allowing this to happen by appending task id to the name if I remember correctly.

Reply:
If that's the case I'll remove the check. The other message from oracle should have been removed before then.

Reply:
We should keep the message from Oracle. That is guarded against race condition which is different from checking the task name from user input.